sim/shaman/windfury_weapon.go

```diff
@@ -40,7 +40,7 @@ func (shaman *Shaman) newWindfuryImbueSpell(isMH bool) *core.Spell {
 		ActionID:    actionID,
 		SpellSchool: core.SpellSchoolPhysical,
 		DefenseType: core.DefenseTypeMelee,
-		ProcMask:    procMask | core.ProcMaskWeaponProc,
+		ProcMask:    procMask,
```

Comment:
I think the windfury weapon trigger also needs to be changed to not proc from procs so that phantoms don't proc it

Reply:
added suppress equip flag